public/assets_embedded.go

```diff
@@ -23,3 +24,18 @@ func Assets() (http.FileSystem, error) {
 
 	return http.FS(fsys), nil
 }
+
+func NewIndexHandler() (http.HandlerFunc, error) {
+	indexPath := filepath.Join("build", "index.html")
+	p, err := assets.ReadFile(indexPath)
+	if err != nil {
+		return nil, err
+	}
+
+	return func(w http.ResponseWriter, r *http.Request) {
+		w.Write(p)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+		}
+	}, nil
```

Comment:
I don't think this is correct, since you can't write a header after writing data. I had to handle the error since the linter was complaining, and panicing seems a bit too much for this case, any suggestions?